os/arch/arm/src/armv8-m/up_exception.S

```diff
@@ -376,12 +408,15 @@ exception_common:
 	.bss
 	.global	g_intstackalloc
 	.global	g_intstackbase
+	.global	g_nestlevel
```

Comment:
is there any limitation on interrupt nesting level?

Reply:
No. There is no limit. But when nestlevel becomes zero, then it means we are returning to process context and so context unstacking will be done.